text/0000-Bundle-Load-API.md

```diff
@@ -67,10 +67,11 @@ template<typename T>
 T GetSymbol(void * handle, const std::string& symname);
 
 ```
-This API will first check if the bundle (from which the symbols are to be fetched) is in START + ACTIVE state (i.e if the bundle is valid or not).
-If not, it will throw an exception with message as "invalid bundle".
+This API will first check if the bundle (from which the symbols are to be fetched) is in START + ACTIVE state.
+If the state is not START + ACTIVE , it will throw an exception with message as "invalid bundle".
```

Comment:
the exception message shouldn't be "invalid bundle". Saying that the bundle isn't started or active should suffice.

Reply:
Ok. I will change it. Thanks.